static/docs/use-cases/data-registry.md

```diff
@@ -1,36 +1,28 @@
 # Data Registry
 
 One of the main uses of <abbr>DVC repositories</abbr> is the
-[versioning of data and model files](/doc/use-cases/data-and-model-files-versioning).
-This is provided by commands such as `dvc add` and `dvc run`, that allow
-tracking of datasets and any other <abbr>data artifacts</abbr>.
-
-With the aim to enable reusability of these versioned artifacts between
-different projects (similar to package management systems, but for data), DVC
-also includes the `dvc get`, `dvc import`, and `dvc update` commands. For
-example, project A may use a data file to begin its data
-[pipeline](/doc/command-reference/pipeline), but project B also requires this
-same file; Instead of
-[adding it](/doc/command-reference/add#example-single-file) it to both projects,
-B can simply import it from A. Furthermore, the version of the data file
-imported to B can be an older iteration than what's currently used in A.
+[versioning of data and model files](/doc/use-cases/data-and-model-files-versioning),
+with commands such as `dvc add`. With the aim to enable reusability of these
+<abbr>data artifacts</abbr> between different projects, DVC also provides the
+`dvc get`, `dvc import`, and `dvc update` commands. This means that a project
+can depend on data from an external <abbr>DVC project</abbr>, **similar to
+package management systems, but for data**.
+
+<!-- Insert diagram image here. -->
 
 Keeping this in mind, we could build a <abbr>DVC project</abbr> dedicated to
```

Comment:
I still stumble on this `could` every time.

Reply:
I don't know what this comment means. Seems like an observation. Resolving.